core/src/main/java/bio/terra/pearl/core/service/export/EnrolleeImportService.java

```diff
@@ -0,0 +1,100 @@
+package bio.terra.pearl.core.service.export;
+
+import bio.terra.pearl.core.model.audit.DataAuditInfo;
+import bio.terra.pearl.core.model.participant.Enrollee;
+import bio.terra.pearl.core.model.participant.ParticipantUser;
+import bio.terra.pearl.core.model.participant.Profile;
+import bio.terra.pearl.core.model.study.StudyEnvironment;
+import bio.terra.pearl.core.model.workflow.HubResponse;
+import bio.terra.pearl.core.service.exception.internal.InternalServerException;
+import bio.terra.pearl.core.service.export.formatters.module.EnrolleeFormatter;
+import bio.terra.pearl.core.service.export.formatters.module.ParticipantUserFormatter;
+import bio.terra.pearl.core.service.participant.ProfileService;
+import bio.terra.pearl.core.service.workflow.EnrollmentService;
+import bio.terra.pearl.core.service.workflow.RegistrationService;
+import com.opencsv.CSVParser;
+import com.opencsv.CSVParserBuilder;
+import com.opencsv.CSVReader;
+import com.opencsv.CSVReaderBuilder;
+import com.opencsv.exceptions.CsvValidationException;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.util.*;
+
+@Service
+public class EnrolleeImportService {
+    /** for now, we only support importing from a specific style of export. */
+    ExportOptions IMPORT_OPTIONS = new ExportOptions(false, true, true, ExportFileFormat.TSV, null);
+
+    private final RegistrationService registrationService;
+    private final EnrollmentService enrollmentService;
+    private final ProfileService profileService;
+
+    public EnrolleeImportService(RegistrationService registrationService, EnrollmentService enrollmentService, ProfileService profileService) {
+        this.registrationService = registrationService;
+        this.enrollmentService = enrollmentService;
+        this.profileService = profileService;
+    }
+
+    @Transactional
+    /**
+     * exports the specified number of enrollees from the given environment
+     * The enrollees will be returned most-recently-created first
+     * */
```

Comment:
nit: "exports" leftover in copy/paste

Reply:
good catch, will fix in later PR to avoid up-the-chain merge conflicts